spec/requests/monthly_report_spec.rb

```diff
@@ -10,6 +10,12 @@
     it { expect(response.body).to match report.user.name }
   end
 
+  describe "root_path" do
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)